internal/cloud/backend_plan.go

```diff
@@ -635,6 +641,22 @@ func (b *Cloud) shouldRenderStructuredRunOutput(run *tfe.Run) (bool, error) {
 	return false, nil
 }
 
+// numericVersion check whether TFE is using Numeric Versioning (X.Y.Z, X.Y.Z-*).
+// starting August, 2025 TFE will use X.Y.Z versioning scheme
+func numericVersion(v string) bool {
+	if strings.HasPrefix(v, "v") || !strings.Contains(v, ".") {
```

Comment:
💅 I believe you can drop `!strings.Contains(v, ".")`, that check is handled by `version.NewVersion()`